test/test-setup.ts

```diff
@@ -5,9 +5,6 @@ import 'reflect-metadata';
 import sinon from 'sinon';
 import CredentialManager from '../src/app-config/credentials';
 import PortUtil from '../src/common/utils/port';
-import PromptUtils from '../src/common/utils/prompt-utils';
-
-PromptUtils.disable_prompts();
```

Comment:
Looks like the `disable_prompts` function can be removed as it's no longer used anywhere